Cargo.toml

```diff
@@ -31,7 +31,7 @@ ignore = "0.4.18"
 clap-verbosity-flag = "2.0.0"
 log = "0.4.17"
 git2 = { version = "0.15.0", default-features = false }
-crates-index = "0.18.9"
+crates-index = { version = "0.18.9", features = ["vendored-openssl"] }
```

Comment:
@epage any concerns with adding this feature? It seems to make the build simpler and more reliable, but I wanted to double-check with you in case there were any downsides I'm not familiar with.